apps/docs/public/templates/search-page.html

```diff
@@ -400,7 +400,7 @@
   <body class="gieds-js">
     <header id="GovieHeader" class="gi-header">
       <div class="gi-header-language-bar">
-        <div class="gi-layout-container">
+        <div class="gi-layout-container-full-width">
```

Comment:
Why are you using `full-width`?
Copy the header from the `basic-page` template, which has the latest requests implemented.

Reply:
Fixed